### PR TITLE
Meet/Zoom Email Clarification

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

This PR adds clarification to email when the user is booking a zoom/meet meeting - that the meeting information will be provided when the event has been confirmed. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: apps/web/lib/emails/templates/organizer-scheduled-email.ts [hunk 1]
```diff
@@ -356,17 +356,17 @@ ${getRichDescription(this.calEvent)}
...
       <p style="height: 6px"></p>
       <div style="line-height: 6px;">
         <p style="color: #494949;">${this.calEvent.organizer.language.translate("where")}</p>
-        <p style="color: #494949; font-weight: 400; line-height: 24px;">${providerName} ${
-        meetingUrl &&
-        `<a href="${meetingUrl}" target="_blank" alt="${this.calEvent.organizer.language.translate(
-          "meeting_url"
-        )}"><img src="${linkIcon()}" width="12px"></img></a>`
-      }</p>
+        <p style="color: #494949; font-weight: 400; line-height: 24px;">${providerName}
```
Comment (sean-brydon) [inline apps/web/lib/emails/templates/organizer-scheduled-email.ts:363]: Formatted to make this more readable

**Comment by CarinaWolli** [comment]
Looks good! Should we maybe also add that to the attendee's email not just the organizer?

File: apps/web/lib/emails/templates/organizer-scheduled-email.ts [hunk 2]
```diff
@@ -356,17 +356,17 @@ ${getRichDescription(this.calEvent)}
...
+        ${
+          meetingUrl &&
+          `<a href="${meetingUrl}" target="_blank" alt="${this.calEvent.organizer.language.translate(
+            "meeting_url"
+          )}"><img src="${linkIcon()}" width="12px"></img></a>`
```
Comment (zomars) [inline apps/web/lib/emails/templates/organizer-scheduled-email.ts:368]:
```suggestion
          )}"><img src="${linkIcon()}" width="12px"/></a>`
```

**Review by zomars — APPROVED** [review]

**Comment by zomars** [comment]
Agreed with @CarinaWolli we need to make sure every party involved receives the clarification.

**Review by emrysal — CHANGES_REQUESTED** [review]
Requesting changes to clarify this should also be implemented in invitee-scheduled-email.ts

**Review by emrysal — APPROVED** [review]

**Pull request merged**